### PR TITLE
Add StackdriverTraceActivityExporter

### PR DESCRIPTION
**Pull request by ian-mi**

### CODE REVIEW

**Comment by cijothomas** [comment]
@ian-mi Thanks!
I will wait to review this until we get CI ready for this repo. Eddy has made a PR adding CI, but its not triggered automatically. 

@MikeGoldsmith @SergeyKanzhelev - can you check if this repo has to do something to enable GH Actions?

File: src/OpenTelemetry.Exporter.Stackdriver/Implementation/ActivityExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// <copyright file="ActivityExtensions.cs" company="OpenTelemetry Authors">
+// Copyright 2020, OpenTelemetry Authors
```
Comment (SergeyKanzhelev) [inline src/OpenTelemetry.Exporter.Stackdriver/Implementation/ActivityExtensions.cs:2]:
```suggestion
// Copyright The OpenTelemetry Authors
```

File: src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```
Comment (SergeyKanzhelev) [inline src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj:3]: it seems that the same needs to happen on a test project. @ian-mi can you pls take a look?

File: src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
     <PackageTags>$(PackageTags);Stackdriver;Google;GCP;distributed-tracing</PackageTags>
     <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="1.1.0-beta02" />
-    <PackageReference Include="Google.Cloud.Trace.V2" Version="1.0.0-beta02" />
+    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="2.1.0" />
+    <PackageReference Include="Google.Cloud.Trace.V2" Version="2.0.0" />
     <PackageReference Include="OpenTelemetry" Version="0.2.0-alpha.404" />
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj:12]: can you update to the latest here?

**Comment by cijothomas** [comment]
@ian-mi - Sorry for ignoring this repo. Was completely occupied with doing beta release!
Are you planning to continue working on this? Let us know you plan!

**Comment by ian-mi** [comment]
Sorry I haven't had time to revisit this yet! I will take a look and see if I can get this ready to merge.

**Comment by eddynaka** [comment]
@ian-mi , in my last pr I removed the reference from nuget and changed everything to activity style. Can you merge master and check if I missed something? Thank you

**Comment by cijothomas** [comment]
@ian-mi Another merge caused some conflicts here. 
This looks good, please resolve conflicts and we can merge.

**Comment by ian-mi** [comment]
This appears to be obsolete now.

**Comment by eddynaka** [comment]
@ian-mi , one of your changes was targetting the new version of Google.Cloud.xxx nugets. Is that required?

File: src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj (hunk 2 above)
Comment (eddynaka) [reply src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj:3]: It looks like with the new version, it only supports net461 or greater. Don't know if this package update is required or not.

File: src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj (hunk 3 above)
Comment (eddynaka) [reply src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj]: @cijothomas , should we update? If yes, we will lose support for net452.

**Pull request closed without merging**